docs/plugins/typescript-react-apollo.md

```diff
@@ -76,6 +76,4 @@ Or if you are using `noNamespaces` option:
   const { data, loading, error } = useTest(...);
 ```
 
-#### `withSubscriptionHooks` (default value: `false`)
-
-This will cause the codegen to add React **Hooks** even for _Subscriptions_. 
+You can specify alternative module that is exports `useQuery` `useMutation` and `useSubscription`. This is useful for further abstraction of some common tasks (eg. error handling). Filepath relative to generated file can be also specified. But if you provide only `true`, it will use `react-apollo-hooks` by default.
```

Comment:
It doesn't feel right to abuse the boolean flag to also have a string value. I would keep it separate, nothing wrong about.